ad_hoc/update_fixtures_contains_specific_string.py

```diff
@@ -0,0 +1,29 @@
+import sys
```

Comment:
We might be able to put this file in the `scripts` directory. Thoughts?

Reply:
Moved, thank you for the suggestion!